src/main/java/com/diffmin/util/SpoonUtil.java

```diff
@@ -73,19 +75,27 @@ public static CtModel buildModel(File file) throws FileNotFoundException {
         return launcher.buildModel();
     }
 
-    /**
-     * Pretty prints the model.
-     *
-     * @param model model to be pretty printed
-     * @return patched program
-     */
-    public static String displayModifiedModel(CtModel model) {
-        CtType<?> firstType = model.getAllTypes().stream().findFirst().get();
-        CtCompilationUnit cu = firstType.getFactory().CompilationUnit().getOrCreate(firstType);
+    public static String prettyPrintModel(CtModel model) {
```

Comment:
Perhaps the name should encompass that we expect only one compilation unit. Maybe `prettyPrintModelWithSingleCompilationUnit`.